src/source/Dtls/Dtls.h

```diff
@@ -55,9 +55,15 @@ typedef struct {
 } DtlsKeyingMaterial, *PDtlsKeyingMaterial;
 
 typedef struct {
-    SSL_CTX *pSslCtx;
+    BOOL created;
     X509 *pCert;
     EVP_PKEY *pKey;
+} DtlsSessionCertificateInfo, *PDtlsSessionCertificateInfo;
```

Comment:
* Do we need to keep the certificates at all? We might be able to just free them right after they are used.
* If you do need to keep a copy around I would copy the user provided certs, then we can drop the `created` member. I could see users calling free after they start the `RtcPeerConnection` (because it is async)

Reply:
Looked at it but it's not a trivial task. The certs are not simple blobs to copy and store around. For now, we need to ensure the certificates are around while the connection is happening

Reply:
We can store the cert fingerprints during the session